Makefile

```diff
@@ -426,8 +426,11 @@ terratest: # Run terratest suite
 		echo "$(RED)Make sure you run the tests against at least two running clusters$(NC)" ;\
 		exit 1;\
 	fi
-	cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS)
-
+	mkdir -p /tmp/terratest && cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS) | tee /tmp/terratest/all.log
+	curl --location --silent --fail --show-error -o terratest_log_parser https://github.com/gruntwork-io/terratest/releases/download/v0.38.8/terratest_log_parser_linux_amd64
+	chmod +x terratest_log_parser
+	sudo mv terratest_log_parser /usr/local/bin
+	terratest_log_parser --testlog /tmp/terratest/all.log --outputdir /tmp/terratest
```

Comment:
locally static `/tmp/terratest` can quickly get messy after multiple invocations. Should we include a randomized string in directory name?

Reply:
As I removed `terratest_log_parser` from Make file, right now logs will be overwriten every time into `all.log` file